code/datums/elements/atmos_sensitive.dm

```diff
@@ -18,6 +18,7 @@
 	var/atom/us = source
 	us.UnregisterSignal(get_turf(us), COMSIG_TURF_EXPOSE)
 	if(us.flags_1 & ATMOS_IS_PROCESSING_1)
+		us.atmos_end(null, null)
```

Comment:
Why are you doing `(null, null)` instead of just `()`?

Reply:
It was an attempt to be clear about the behavior of the element, but in retrospect it was unneeded